scripts/build-dependencies.sh

```diff
@@ -0,0 +1,22 @@
+if [ ! -d ./closure-library/.git ]; then
+    rm -rf closure-library
+    git clone --branch v20180910 https://github.com/google/closure-library/
```

Comment:
[Question] Do we need to clone the master branch or this specific branch `v20180910`?

Reply:
latest working branch for now